jwst/transforms/models.py

```diff
@@ -1147,15 +1148,16 @@ def evaluate(self, x, y, x0, y0, order):
         xmodel = self.xmodels[iorder]
         ymodel = self.ymodels[iorder]
         lmodel = self.lmodels[iorder]
-        dx = xmodel[0](x00, y00) + t * xmodel[1](x00, y00)
-        dy = ymodel[0](x00, y00) + t * ymodel[1](x00, y00)
+        dx = xmodel[0](x00, y00) + t * xmodel[1](x00, y00) + t**2 * xmodel[2](x00, y00)
+        dy = ymodel[0](x00, y00) + t * ymodel[1](x00, y00) + t**2 * ymodel[2](x00, y00)
 
         if self.theta != 0.0:
             rotate = Rotation2D(self.theta)
             dx, dy = rotate(dx, dy)
+        # so = np.argsort(dy)
```

Comment:
These commented lines appear to be duplicates of the active lines below.

Reply:
removed